src/main/java/Floda.java

```diff
@@ -0,0 +1,83 @@
+import java.util.Scanner;
+
+public class Floda {
+    public static void main(String[] args) {
+        String name = "Floda";
+        System.out.println("Hello! I'm " + name);
+        Task[] list = new Task[100];
+        int taskCounter = 0;
+        Scanner scanner = new Scanner(System.in);
+        String line;
+        System.out.println("I can keep track of a to-do list for you! Just type what you want to add to the list.");
+        while (!"bye".equals((line = scanner.nextLine()))) {
```

Comment:
It would improve readability if you directly assigned String line = scanner.nextLine() in the previous line instead of assigning it inside the conditional for the while loop. Then you could read in a new line of input at the end of every if else block.